tests/spec/examples.py

```diff
@@ -796,6 +777,50 @@ def it_runs_before_starting_a_service(self):
         )
 
 
+class DescribePostStartHook(object):
+
+    @pytest.yield_fixture
+    def service_name(self):
+        yield 'post-start-hook'
+
+    @pytest.mark.usefixtures('in_example_dir')
+    def it_runs_after_all_services_have_started(self):
+        assert_command(
+            ('pgctl', 'start', 'A'),
+            '',
+            '''\
+[pgctl] Starting: A
+[pgctl] Started: A
+''',
+            0,
+            norm=norm.pgctl,
+        )
+        assert_command(
+            ('pgctl', 'start', 'B'),
+            '',
+            '''\
+[pgctl] Starting: B
+[pgctl] Started: B
+hello, i am a post-start script
```

Comment:
@bukzor: suppose a playground has two services (A, B) and a post start hook. Are you saying that you expect behavior like this:
```
$ pgctl start B
...
post start script says hi

$ pgctl start A
...
post start script says hi
```
If we're keeping with the pre-start convention, that makes a lot of sense. It would then be up to the script to figure out if the right set of services are up or not (@smallredbean can we use pgctl status from the script or?). Overall that sounds like a more flexible approach.

The flip side of that argument is that post-start feels like something that runs exactly once after the state transition Stopped (*) -> Started (*), where the state transition is for the **entire** playground. The trouble with that is it is just a special case of the other implementation strategy.